dao/src/main/java/com/epam/esm/dao/DBOperationCRD.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.esm.dao;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface DBOperationCRD<T> {
+
+    Long createFromEntity(T entity);

```

Comment:
Do you support other method for creation resources? Not from entity?

Reply:
no, i thought it would be nice to keep method name more consist. but, really there's no need in it. renamed